alpine/alpine-unfixed/alpine.go

```diff
@@ -0,0 +1,129 @@
+package alpineunfix
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/xerrors"
+
+	"github.com/spf13/afero"
+
+	"github.com/aquasecurity/vuln-list-update/utils"
+)
+
+const (
+	alpineDir = "alpine-unfix"
+	filePath  = "alpine/alpine-unfixed/all.tar.gz"
+	secFixUrl = "https://aquasecurity.github.io/secfixes-tracker/all.tar.gz"
+)
+
+type Updater struct {
+	vulnListDir      string
+	appFs            afero.Fs
+	baseURL          string
+	fileDownloadPath string
+	retry            int
+}
+
+func NewUpdater() *Updater {
+	updater := &Updater{
+		vulnListDir:      utils.VulnListDir(),
+		appFs:            afero.NewOsFs(),
+		baseURL:          secFixUrl,
+		fileDownloadPath: filePath,
+	}
+	return updater
+}
+func (u Updater) Update() (err error) {
+	dir := filepath.Join(u.vulnListDir, alpineDir)
+	log.Printf("Remove Alpine directory %s", dir)
+	if err := u.appFs.RemoveAll(dir); err != nil {
+		return xerrors.Errorf("failed to remove Alpine directory: %w", err)
+	}
+	if err := u.appFs.MkdirAll(dir, 0755); err != nil {
+		return err
+	}
+
+	log.Println("Fetching Alpine unfix data...")
+	err = utils.DownloadFile(u.fileDownloadPath, u.baseURL)
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(u.fileDownloadPath) // nolint: errcheck
+
+	err = u.ExtractTarGz(dir)
+	if err != nil {
+		return err
+	}
+	return
+}
+
+func (u Updater) ExtractTarGz(saveDir string) error {
```

Comment:
Could you rewrite this function by using https://github.com/aquasecurity/vuln-list-update/pull/105?